doc/api/path.md

```diff
@@ -431,6 +431,39 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+*Note*: Be careful when using the following functions directly on Windows.
+
+### path.posix.resolve([...path])
+If after processing all given `path` segments an absolute path has not yet
+been generated, `path.posix.resolve` will throw [`UNSUPPORTED_PLATFORM`] error.
+
```

Comment:
Perhaps:

```
*Note*: Care should be taken when using `path.posix.resolve()` on Windows due to ...
```

Then include a short snippet about why...

Same with the `path.posix.relative()` below.